.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "astropy-helpers"]
 	path = astropy_helpers
 	url = https://github.com/astropy/astropy-helpers.git
+[submodule "relic"]
+	path = relic
+	url = https://github.com/jhunkeler/relic.git
```

Comment:
So, is it pretty sure that you are not going to move this under `spacetelescope`?